src/templates/next/components/Heading/Heading.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { Meta, StoryFn, StoryObj } from "@storybook/react"
+import Heading from "./Heading"
+import { HeadingProps } from "~/common"
+import { HeadingLevels } from "~/common/Heading"
+
+export default {
+  title: "Next/Components/Heading",
+  component: Heading,
+  argTypes: {},
+  parameters: {
+    themes: {
+      themeOverride: "Isomer Next",
+    },
+  },
+} as Meta
+
+// Template for stories
+
+const Headings = () => {
+  return (
+    <div>
+      {HeadingLevels.map((level) => {
+        return (
+          <div className="mb-4">
+            <Heading level={level} content={`This is a heading-${level}`} />
+          </div>
+        )
+      })}
+    </div>
+  )
+}
+
+export const ColorsAndVariants: StoryObj<HeadingProps> = {
+  render: () => <Headings />,
+}
```

Comment:
nit: quite cool but then lmao this makes it not possible to play with the props at the bottom of the storybook :(

Reply:
it's still possible 😄 , I did it for Button

https://github.com/isomerpages/isomer-components/blob/main/src/templates/next/components/Button/Button.stories.tsx

Reply:
I cannot get eh why ah lol. There's no box for me to change the contents.

![Screenshot 2024-03-18 at 21 59 35](https://github.com/isomerpages/isomer-components/assets/27919917/c4b49a82-6789-4728-af9d-1c073df578b7)

Reply:
oh nvm I thought you were saying that cannot add other stories with different props.
yea cannot tweak props in the rendered story